app/utils/database.py

```diff
@@ -87,6 +87,12 @@ class BaseModel(Model):
     This class sets the database to use the postgres database.
     """
 
+    def as_dict(self):
+        """
+        Returns the objects key/value pair as a dictionary.
+        """
+        return self.__dict__['__data__']
```

Comment:
Is this `__data__` set by the SQL lib? Is this the same as `self.__data__`? Is there no library function that can be used instead? `self.get_data()` or something.

Reply:
I honestly didn't find any. Here're the docs: http://docs.peewee-orm.com/en/latest/peewee/api.html#Model - and as far as I can tell you're "supposed" to handle the result data as objects, but those aren't json serializable...